### ISSUE TITLE
stub is broken when localize guard object has released in a different order

### ISSUE THREAD

**Issue by shibayu36** [astj/p5-Test-WWW-Stub#19]
Following code doesn't run as expected;

```perl
my $ua = LWP::UserAgent->new;

my $g1 = Test::WWW::Stub->register('https://example.com/OVERRIDE' => [ 500, [], [] ]);
say $ua->get('https://example.com/OVERRIDE')->code;
my $g2 = Test::WWW::Stub->register('https://example.com/OVERRIDE' => [ 400, [], [] ]);
say $ua->get('https://example.com/OVERRIDE')->code;
$g1 = undef;
say $ua->get('https://example.com/OVERRIDE')->code;
$g2 = undef;
say $ua->get('https://example.com/OVERRIDE')->code;
```

The result is below.
```
500
400
Unexpected external access: GET https://example.com/OVERRIDE
499
500
```

**Comment by astj** [comment]
@shibayu36 Sorry for kept you waiting for so looooooong. I fixed at #21 and released as 0.09.

**Comment by shibayu36** [comment]
Thanks for your release!!
